test/Microsoft.ML.Benchmarks/Helpers/EmptyWriter.cs

```diff
@@ -7,16 +7,12 @@
 
 namespace Microsoft.ML.Benchmarks
 {
-    internal class Helpers
-    {
-        public static string DatasetNotFound = "Could not find {0} Please ensure you have run 'build.cmd -- /t:DownloadExternalTestFiles /p:IncludeBenchmarkData=true' from the root";
-    }
-
     // Adding this class to not print anything to the console.
```

Comment:
@Anipik or @adamsitnik - FYI - whoever is in the benchmark tests again. A new class that just came in is the `LocalEnvironment`, which doesn't print to the console by default. We could remove this class and switch our environment from `ConsoleEnvironment` to `LocalEnvironment`. That will work around the issue as well, and is simpler.